src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -233,6 +233,24 @@ export default function GamesSubmenu({
     isInstalled &&
     !isThirdPartyManaged
 
+  const hasWine = !is.win && !is.native
+
+  const onBrowseFiles = useCallback(() => {
+    const path = gameInfo.install.install_path || gameInfo.folder_name
```

Comment:
not sure if `gameInfo.folder_name` would help here as a fallback, this is just the folder name, not a path that can be opened

Reply:
I've copied the way it seems to be done here:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/3f7033e7b37f4ccddec73f5187a98c9956909920/src/frontend/screens/Game/GamePage/components/InstalledInfo.tsx#L55
Is this incorrect?

Reply:
hmmm that's weird cause we can't really open just a folder name, it wouldn't be a path, maybe when that was first implemented it was not meant to be clicked, I don't really know, I do think it won't work (you can try it, just remove the `gameInfo.instlal.install_path || ` part of that assignment and click the `Browse Files` link, I imagine it won't open at all)

Reply:
You're right about `folder_name` not leading to a valid path in the case of a GOG game:
![image](https://github.com/user-attachments/assets/af995fea-5f50-4f62-bcea-e7eb682a1c26)

However it looks like it is a valid path for sideloaded games, which do not seem to list an install path (other than the executable):
![image](https://github.com/user-attachments/assets/baf5d21d-6ee3-434e-8a5f-9a049a8d3c9a)

So it looks like the existing code from `InstalledInfo.tsx` is handling this logic correctly, shouldn't we do the same thing here?

Reply:
oh ok, I didn't think it was something for the sideloaded games, yes we should do that in both places then to account for that case